src/app/combo/combo.sample.ts

```diff
@@ -410,4 +418,14 @@ export class ComboSampleComponent implements OnInit, AfterViewInit {
             }
         );
     }
+
+    public openOnClearSelectionChanging(event: ISimpleComboSelectionChangingEventArgs){
+        if(!event.newSelection){
+            this.openOnClearSimpleComboEvent.open();
+        }   
+    }
+
+    public openSimpleComboTemplate(){
```

Comment:
No need for this handler - we can put the logic that opens the combo directly in the template.